python_backend/api.py

```
@@ -141,13 +541,236 @@ async def generate_translated_pptx(
+            max_slides=max_slides,
+            dpi=dpi,
+            max_dimension=max_dimension)
+
+        # generate_slide_previews モジュールをインポート
+        try:
+            from generate_slide_previews import generate_slide_previews
+            preview_paths = generate_slide_previews(PathLib(input_path), PathLib(output_dir))
+        except ImportError:
+            # フォールバック: subprocessで実行
+            logger.warning("Direct import failed, using subprocess")
+            result = subprocess.run(
```

Ruff (0.14.10)

`subprocess` call: check for execution of untrusted input

(S603)